shared/logutil/logutil.go

```diff
@@ -23,3 +26,19 @@ func ConfigurePersistentLogging(logFileName string) error {
 	logrus.Info("File logging initialized")
 	return nil
 }
+
+// CountdownToGenesis logs the time remains until genesis time
```

Comment:
```suggestion
// CountdownToGenesis logs the time remaining until the specified genesis time.
```

Reply:
Are you planning to address this? I see your :+1:

Reply:
Oh sorry, when I stopped working on it forgot to push that review.